proxy_agent_extension/src/main.rs

```diff
@@ -56,8 +60,8 @@ fn main() {
     if let Some(command) = cli.command {
         // extension commands
         let config_seq_no =
-            env::var("ConfigSequenceNumber").unwrap_or_else(|_e| "no seq no".to_string());
-        handler_main::program_start(command, Some(config_seq_no));
+            env::var(CONFIG_SEQ_NO_ENV_VAR).unwrap_or_else(|_e| "no seq no".to_string());
+        handler_main::program_start(command, config_seq_no);
```

Comment:
removing the Option field for config_seq_no may not have any consequences, but how does it help besides removing some errors which could instead be replaced with custom errors?